sysdig/internal/client/v2/sysdig.go

```diff
@@ -43,6 +43,7 @@ type SysdigSecure interface {
 	VulnerabilityExceptionListInterface
 	VulnerabilityExceptionInterface
 	CloudAccountSecureInterface
+	CloudauthAccountSecureInterface
```

Comment:
Same here, let's not have Cloudauth here.

Reply:
same here.